src/user/dto/user.dto.ts

```diff
@@ -0,0 +1,54 @@
+import { ApiProperty, PartialType } from '@nestjs/swagger'
+import { Exclude, Expose, Type } from 'class-transformer'
+import { IsDate, IsEnum, IsNotEmpty, IsNumber, IsOptional, IsPhoneNumber } from 'class-validator'
+import { PersonalInfo } from '../user.interface'
+
+export class CreateUserDto extends PersonalInfo {
+  @ApiProperty()
+  @IsNotEmpty()
+  @IsPhoneNumber('TH')
```

Comment:
This 'TH' means only thai phone is except? if yes, I think this should be changed

Reply:
I'll change it to global phones but frontend will have to specify region code.